apps/prairielearn/package.json

```diff
@@ -175,7 +175,7 @@
     "three": "0.160.1",
     "tmp": "^0.2.3",
     "tmp-promise": "^3.0.3",
-    "tom-select": "^2.3.1",
+    "tom-select": "^2.4.0",
```

Comment:
Let's bump this to 2.4.1, I released that this week!

Reply:
Nice! Bumped to 2.4.1 in [98f2dd5](https://github.com/PrairieLearn/PrairieLearn/pull/10868/commits/98f2dd558c3f286228530e2a7899b8db990394be).